src/modules/markets/selectors/markets-totals.js

```diff
@@ -1,30 +1,23 @@
 import { createSelector } from 'reselect';
 import store from 'src/store';
 import selectAllMarkets from 'modules/markets/selectors/markets-all';
-import { selectFilteredMarkets } from 'modules/markets/selectors/markets-filtered';
-import { selectUnpaginatedMarkets } from 'modules/markets/selectors/markets-unpaginated';
-import { selectFavoriteMarkets } from 'modules/markets/selectors/markets-favorite';
 
 export default function () {
   return selectMarketsTotals(store.getState());
 }
 
 export const selectMarketsTotals = createSelector(
   selectAllMarkets,
-  selectFilteredMarkets,
-  selectUnpaginatedMarkets,
-  selectFavoriteMarkets,
-  (allMarkets, filteredMarkets, unpaginatedMarkets, favoriteMarkets) => {
+  (allMarkets) => {
```

Comment:
At a glance, it appears that filtered/unpaginated/favoriteMarkets are no longer memoized; is this correct?  If so, are you concerned about a performance hit?  The markets selectors can get pretty large.